src/shared/schemas.ts

```diff
@@ -21,6 +25,9 @@ import { CloudFormation } from './cloudformation/cloudformation'
 
 const GOFORMATION_MANIFEST_URL = 'https://api.github.com/repos/awslabs/goformation/releases/latest'
 const SCHEMA_PREFIX = `${AWS_SCHEME}://`
+const BUILD_SPEC_HOSTED_FILES_PATH = '/CodeBuild/buildspec/buildspec-standalone.schema.json'
+const BUILDSPEC_CLOUDFRONT_URL = 'https://d3rrggjwfhwld2.cloudfront.net' + BUILD_SPEC_HOSTED_FILES_PATH
+const BUILDSPEC_S3_FALLBACK_URL = 'https://aws-vs-toolkit.s3.amazonaws.com' + BUILD_SPEC_HOSTED_FILES_PATH
```

Comment:
There are not many uppercase names here, let's take the opportunity to discontinue that practice here. Just use normal names. There is nothing so special about constants that they need to be uppercase.

```suggestion
const buildspecS3FallbackUrl = 'https://aws-vs-toolkit.s3.amazonaws.com' + BUILD_SPEC_HOSTED_FILES_PATH
```